galaxy/templates/jobs-init.yaml

```diff
@@ -107,14 +180,38 @@ spec:
               mountPath: /galaxy/server/config/mutable/
               subPath: config
             {{- if .Values.cvmfs.enabled }}
-            - name: cvmfs-gxy-cloud
-              mountPath: {{ .Values.cvmfs.cloud.mountPath }}
-            - name: cvmfs-gxy-data
-              mountPath: {{ .Values.cvmfs.data.mountPath }}
+            {{- range $key, $entry := .Values.cvmfs.galaxyPersistentVolumeClaims }}
+            - name: cvmfs-gxy-{{ $key }}
+              mountPath: {{ $entry.mountPath }}
+            {{- end -}}
             {{- end }}
             {{- if .Values.extraVolumeMounts }}
             {{- .Values.extraVolumeMounts | toYaml | nindent 12 }}
             {{- end }}
+        - name: {{ .Chart.Name }}-init-cloud-repo
+          image: alpine:3.7
+          command: ['sh', '-c', 'wget -O startup.tar.gz {{ .Values.initJob.externalConfs.archives.startup }} && tar -xvf startup.tar.gz > /dev/null && echo "Done" > /galaxy/server/config/mutable/init_clone_done_{{.Release.Revision}}']
```

Comment:
It seems like all archives are downloaded regardless of the setting? Should this be conditional instead?

Reply:
No. We need all of them. The startup one needs to contain at least just the XMLs which is enough for the handlers to load <2Mb). The second one needs to contain at least all wrappers and scripts, and the last one can container anything (mostly adding tool test data).